deployment/aks-periscope.yaml

```diff
@@ -92,13 +92,20 @@ spec:
             name: collectors-config
         - secretRef:
             name: azureblob-secret
+        volumeMounts:
```

Comment:
📓 Side note: this needs to plan separately with the release somehow, because of the effect it has to the current feature set.